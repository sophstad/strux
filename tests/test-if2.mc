void main() {
	if (true) {
		print("it's true!");
		print("it's still true here!");
	}
}
