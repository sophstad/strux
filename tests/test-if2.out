it's true!
it's still true here!